src/main/java/org/betonquest/betonquest/events/PickRandomEvent.java

```diff
@@ -12,6 +12,7 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.concurrent.ThreadLocalRandom;
```

Comment:
I would not use ThreadLocalRandom, as it has a very bad randomness, much better would be `ThreadLocal<Random>`, but actually `Random` is thread-safe (but not very efficient with many threads).

Additionally there is a new interface [`RandomGenerator`](https://docs.oracle.com/en/java/javase/19/docs/api/java.base/java/util/random/RandomGenerator.html) since Java 17 with improved implementations available in [`java.util.random`](https://docs.oracle.com/en/java/javase/19/docs/api/java.base/java/util/random/package-summary.html) (but not thread-safe, see the linked documentation for more info).